### ISSUE TITLE
[Global] Missing header structure

### ISSUE THREAD

**Issue by svinkle** [CovidShield/mobile#7]
## Issue summary

View titles are missing a `header` role. Headers are one of the primary methods of navigation for screen reader user. Without a this role, a user may have difficulty understanding the content structure which could lead to a frustrating user experience.

<img width="369" alt="Screen Shot 2020-05-22 at 12 23 11 PM" src="https://user-images.githubusercontent.com/1392632/82688553-12208480-9c27-11ea-970f-b9fce38253bb.png">

#### TSX

```tsx
// NoExposureView.tsx

<Text
  variant="bodyTitle"
  color="bodyText"
  marginBottom="l"
  textAlign="center"
>
  {i18n.translate('Home.NoExposureDetected')}
</Text>
```

## Steps to reproduce

1. Install the CovidShield `apk` on an Android device
2. Load the app
3. Go to the first screen with the "Covid Shield is ON" card
4. Enable [TalkBack screen reader](https://support.google.com/accessibility/android/answer/6007100?hl=en)
5. Swipe right to navigate to the header text

### Expected

- Header text to be announced as a header.

### Actual

- Header role is missing.

## Recommendation

1. For each `variant="bodyTitle"` content node, include the [`accessibilityRole`](https://reactnative.dev/docs/accessibility#accessibilitylabel-android-ios) prop.
2. Set the value to `header`.

#### TSX

```tsx
// NoExposureView.tsx

<Text
  variant="bodyTitle"
  color="bodyText"
  marginBottom="l"
  textAlign="center"
  accessibilityRole="header"
>
  {i18n.translate('Home.NoExposureDetected')}
</Text>
```

## Specifications

- **Component:** CovidShield [Global]
- **WCAG Principle:** Operable
- **WCAG SC:** 2.4.6 Headings and Labels
- **Severity:** Medium
- **Effort:** Low

**Comment by AbubakarSambo** [comment]
Hi @svinkle , Can you see if this is okay: https://github.com/CovidShield/mobile/pull/18/files

**Comment by jordanAtShopify** [comment]
Fixed by https://github.com/CovidShield/mobile/pull/18
